src/language-html/utils/index.js

```diff
@@ -638,6 +638,10 @@ function getTextValueParts(node, value = node.value) {
     : getDocParts(join(line, splitByHtmlWhitespace(value)));
 }
 
+function isVueScriptTag(node, options) {
+  return isVueSfcBlock(node, options) && isScriptLikeTag(node);
```

Comment:
`isScriptLikeTag` ?https://github.com/prettier/prettier/blob/35b2600c0d3b7d295dfc31ce0877c030bf8af51b/src/language-html/utils/index.js#L112-L121,  

```suggestion
  return isVueSfcBlock(node, options) && node.name === "script"
```